code/__HELPERS/globalaccess.dm

```diff
@@ -3272,8 +3272,8 @@
 			global.trade_shuttle=newval
 		if("vox_shuttle")
 			global.vox_shuttle=newval
-		if("voxresearch_shuttle")
-			global.voxresearch_shuttle=newval
+		//if("voxresearch_shuttle")
+		//	global.voxresearch_shuttle=newval
```

Comment:
Just delete these rather than comment them out. File's big enough as it is.

Reply:
>manually editing globalaccess

Reply:
> Can't get the python script to work if your byond is installed anywhere but the C: Drive

Fucking PATH

Reply:
just `cd` into your byond\bin
smh